Cargo.toml

```diff
@@ -1,13 +1,14 @@
 [package]
 name = "colorimetry"
-version = "0.0.3"
+version = "0.0.4"
 edition = "2021"
 rust-version = "1.80"
 description = "Rust library for color modeling with JavaScript/WASM interfaces"
 authors = ["Gerard Harbers", "Harbers Bik LLC"]
 repository = "https://github.com/harbik/colorimetry"
 keywords = ["CIE", "CIELAB", "CIECAM16", "Colorimetry"]
 license = "MIT OR Apache-2.0"
+readme = "README.md"
```

Comment:
Nitpick. But this is the default, so it does not need to be specified. From [the docs](https://doc.rust-lang.org/cargo/reference/manifest.html#the-readme-field):
> If no value is specified for this field, and a file named README.md, README.txt or README exists in the package root, then the name of that file will be used